examples/06_benchmarks/movielens.ipynb

```diff
@@ -76,18 +76,18 @@
      "name": "stdout",
```

Comment:
<p>it's interesting that the times of lightgcn have changed so much. The original was computed with TF 1.15.2 vs 1.15.4 on this one. Not sure why</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1365/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
I guess the time difference because of different hardware specs?

My server specs:
- CPU Intel Xeon E5-2650 v4 @ 2.20GHz
- GPU Nvidia Tesla P100 (16GB)
- RAM 256GB

Reply:
ok great, I think I'm going to merge this PR and then we might probably run it again with one of the Azure machines, to have standard resutls.